src/installer/pkg/projects/Microsoft.DotNet.ILCompiler.LLVM/Microsoft.DotNet.ILCompiler.LLVM.pkgproj

```diff
@@ -2,56 +2,51 @@
-        <TargetPath>tools</TargetPath>
-      </File>
-      <File Include="$(CoreCLRAotSdkDir)*">
-        <TargetPath>sdk</TargetPath>
-      </File>
+    <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
+      <File Include="$(CoreCLRBuildIntegrationDir)*" TargetPath="build" />
+      <File Include="$(CoreCLRILCompilerDir)netstandard\*" TargetPath="tools/netstandard" />
+
+      <!-- TODO-LLVM-Upstream: https://github.com/dotnet/runtimelab/issues/2576 -->
+      <File Include="$(ArtifactsBinDir)Microsoft.Interop.SourceGeneration\$(Configuration)\netstandard2.0\*" TargetPath="analyzers/dotnet/cs" />
+      <File Include="$(ArtifactsBinDir)JSImportGenerator\$(Configuration)\netstandard2.0\*" TargetPath="analyzers/dotnet/cs" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(PackageTargetRuntime)' != '' and '$(BuildingIlcHostPackage)' == 'true'">
+      <File Include="$(CoreCLRILCompilerDir)*" TargetPath="tools" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(PackageTargetRuntime)' != '' and '$(BuildingIlcHostPackage)' != 'true'">
+      <File Include="@(LibrariesRuntimeFiles)" TargetPath="runtimes/$(PackageTargetRuntime)/lib/%(LibrariesRuntimeFiles.NativeSubDirectory)" 
+        Condition="'%(LibrariesRuntimeFiles.IsNative)' != 'true'" />
+      <File Include="@(LibrariesRuntimeFiles)" TargetPath="runtimes/$(PackageTargetRuntime)/native/%(LibrariesRuntimeFiles.NativeSubDirectory)" 
+        Condition="'%(LibrariesRuntimeFiles.IsNative)' == 'true'" />
+      <File Include="$(CoreCLRAotSdkDir)*" TargetPath="runtimes/$(PackageTargetRuntime)/native/%(LibrariesRuntimeFiles.NativeSubDirectory)" />
+      <File Include="$(MibcOptimizationDataDir)/$(TargetOS)/$(TargetArchitecture)/**/*.mibc" TargetPath="mibc" />
+
       <DotnetJsCoreCLRAotSdkExtension Include="ts;js;map;json" />
-      <File Include="$(CoreCLRAotSdkDir)dotnetjs/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="sdk/dotnetjs" />
-      <File Include="$(CoreCLRAotSdkDir)dotnetjs/src/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="sdk/dotnetjs/src" />
-      <File Include="$(CoreCLRAotSdkDir)dotnetjs/src/es6/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="sdk/dotnetjs/src/es6" />
-      <File Include="$(MibcOptimizationDataDir)/$(TargetOS)/$(TargetArchitecture)/**/*.mibc">
-        <TargetPath>mibc</TargetPath>
-      </File>
+      <File Include="$(CoreCLRAotSdkDir)dotnetjs/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="runtimes/$(PackageTargetRuntime)/native/%(LibrariesRuntimeFiles.NativeSubDirectory)" />
+      <File Include="$(CoreCLRAotSdkDir)dotnetjs/src/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="runtimes/$(PackageTargetRuntime)/native/%(LibrariesRuntimeFiles.NativeSubDirectory)" />
+      <File Include="$(CoreCLRAotSdkDir)dotnetjs/src/es6/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="runtimes/$(PackageTargetRuntime)/native/%(LibrariesRuntimeFiles.NativeSubDirectory)" />
```

Comment:
This does not look right. The JS targets expect the file structure that mirrors the source.
```suggestion
      <File Include="$(CoreCLRAotSdkDir)dotnetjs/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="runtimes/$(PackageTargetRuntime)/native/dotnetjs" />
      <File Include="$(CoreCLRAotSdkDir)dotnetjs/src/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="runtimes/$(PackageTargetRuntime)/native/dotnetjs/src" />
      <File Include="$(CoreCLRAotSdkDir)dotnetjs/src/es6/*.%(DotnetJsCoreCLRAotSdkExtension.Identity)" TargetPath="runtimes/$(PackageTargetRuntime)/native/dotnetjs/src/es6" />
```